amazon-chime-sdk-js/src/insertablestreamworker/InsertableStreamWorker.ts

```diff
@@ -0,0 +1,167 @@
+
+export default class InsertableStreamWorker {
+  private currentCryptoKey: string;
+  private useCryptoOffset: boolean = true;
+  private currentKeyIdentifier: number = 0;
+  private scount = 0;
+  private rcount = 0;
+
+
+  private readonly frameTypeToCryptoOffset = {
+    key: 10,
+    delta: 3,
+    undefined: 1,
+  };
+
+  dump(encodedFrame: any, direction: any, max = 16): void {
+    console.log('*** dump::direction:', direction)
+    const data = new Uint8Array(encodedFrame.data);
+    let bytes = '';
+    for (let j = 0; j < data.length && j < max; j++) {
+      bytes += (data[j] < 16 ? '0' : '') + data[j].toString(16) + ' ';
+    }
+    const metadata = encodedFrame.getMetadata();
+    console.log(
+      performance.now().toFixed(2),
+      direction,
+      bytes.trim(),
+      'len=' + encodedFrame.data.byteLength,
+      'type=' + (encodedFrame.type || 'audio'),
+      'ts=' + encodedFrame.timestamp,
+      'ssrc=' + metadata.synchronizationSource,
+      'pt=' + (metadata.payloadType || '(unknown)'),
+      'mimeType=' + (metadata.mimeType || '(unknown)')
+    );
+  }
+
+  encodeFunction(encodedFrame: any, controller: any): void{
+    console.log('*** encodeFunction, this.scount:', this.scount)
+    if (this.scount++ < 30) {
+      // dump the first 30 packets.
+      this.dump(encodedFrame, 'send');
+    }
+    if (this.currentCryptoKey) {
```

Comment:
It looks like we don't set currentCryptoKey anywhere? If it is not set, then we it is not doing any encryption and decryption. It would great to set it and confirm the encode/decode functions work.